lib/matplotlib/tests/test_pickle.py

```diff
@@ -236,7 +236,7 @@ def test_grid():
     fig = manager.canvas.figure
     ax = fig.add_subplot(1, 1, 1)
     ax.grid()
-    # Drawing the grid triggers instance methods to be attached
+    # Drawing the grid used to trigger instance methods to be attached
     # to the Line2D object (_lineFunc).
```

Comment:
Should be juste get rid of the whole test?  Given that it is exactly there to test the workaround that this PR removes.